src/main/java/seedu/duke/ui/Ui.java

```diff
@@ -200,6 +200,15 @@ public void printTaskList(TaskList taskList) {
         System.out.println(LINE);
     }
 
+    /**
+     * Displays a message indicating that there are no tasks in the list.
+     */
+    public void printEmptyTaskList() {
+        System.out.print(LINE);
+        System.out.println(PADDING + "There are no tasks in the list.");
+        System.out.println(LINE);
+    }

```

Comment:
Might be better to just use `printMessage()` instead? Since the format is the same.